plugins/by-name/perfanno/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, ... }:
+
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "perfanno";
+  packPathName = "perfanno.nvim";
+  package = "perfanno-nvim";
+  description = "NeoVim lua plugin that annotates source code with profiling information from perf, LuaJIT, or other profilers.";
+  # TODO: Remove after https://github.com/NixOS/nixpkgs/pull/438707
+  url = "https://github.com/t-troebst/perfanno.nvim";
```

Comment:
https://github.com/NixOS/nixpkgs/pull/438707 is in all unstable channels now, and _should_ be part of the revision we've updated to.

It should be safe to remove the explicit URL now:

```suggestion
```

_Note to self: should we throw when `url` is explicitly defined the same as the package's `meta.homepage`?_